neo/rawio/baserawio.py

```diff
@@ -1284,6 +1298,7 @@ def _get_signal_size(self, block_index: int, seg_index: int, stream_index: int):
 
         All channels indexed must have the same size and t_start.
         """
+        # must NOT be implemented if has_buffer_description_api=True
```

Comment:
Will be ignored?

Reply:
No this is done in the BaseRawWithBufferApiIO sub class